.github/workflows/build.yaml

```diff
@@ -35,7 +35,9 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - macos-latest
+          - macos-15 # ARM
+          - macos-14 # ARM
```

Comment:
Why is two ARMs useful?

Reply:
Mostly just a hedge against Apple breaking/changing any of the OS APIs we're interacting with.